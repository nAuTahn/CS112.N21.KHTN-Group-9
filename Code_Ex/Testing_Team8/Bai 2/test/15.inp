114
2 9981 2791 4490
1 9451 3692 1813
2 5957 3071 3542
1 8186 4509 6064
1 6931 257 1358
1 8055 1684 3388
1 6438 5389 5597
1 9674 4367 1532
2 7731 4325 5067
1 9599 2766 2656
2 7247 5718 4891
2 5807 1383 2844
2 8267 3960 7141
1 6746 3176 4074
2 8473 1331 5324
1 7168 366 783
1 7915 3009 2460
2 5882 4202 1001
2 8834 5461 6403
1 10000 8486 6230
1 8196 673 6731
2 8790 4503 2348
2 6568 3545 6016
1 9126 128 5698
1 9234 7215 8723
2 7797 5109 2089
2 4380 1191 2577
1 8367 5613 7988
2 6778 287 4976
1 9358 396 7079
2 9329 3338 4734
2 8128 1343 1987
2 8735 1160 6201
1 5519 1081 4060
2 7363 3391 4964
1 6466 5927 5703
2 8280 5167 1470
1 1419 403 285
1 8953 7834 1605
1 7552 1178 4091
1 6681 2302 3800
1 7329 3917 5130
2 9678 1241 9095
2 9860 5616 4623
2 8684 4746 5522
2 8674 418 7161
1 4934 831 4558
2 7461 2786 2812
1 9020 7861 485
1 7737 3003 4119
2 8966 511 1054
1 5260 2007 4149
2 6017 5017 1973
2 7966 7895 4005
1 8563 51 1790
2 7305 2504 746
1 8202 1561 485
1 7492 2093 3132
2 9561 4329 7261
2 9353 4260 6412
2 8508 6672 5788
1 9456 3360 2834
2 9916 4500 9446
2 7861 6532 1059
1 4825 2958 4172
1 4379 2074 3571
2 7488 1410 1861
1 7987 506 7113
2 8090 2511 7409
1 8315 377 7309
1 9361 1597 8666
1 6140 1541 4624
1 8703 7551 4393
2 9328 7649 5560
1 9503 2997 1898
1 8724 1701 6422
2 3850 477 1532
1 4836 808 3081
2 8878 44 295
2 5626 3998 2858
2 9695 907 3205
1 8124 125 6207
1 6731 3367 4409
1 5169 340 1639
2 9813 861 9661
2 7302 2154 404
1 6940 5807 6352
2 9285 8995 3600
1 8622 3658 615
2 7498 1311 2914
2 6402 846 1559
1 9028 2942 888
1 4242 2318 1566
1 8939 1260 5479
2 2958 702 2798
2 7664 4184 1243
1 9091 3558 4236
1 6743 847 1217
2 9498 975 8112
2 9772 3180 8801
1 7392 6857 2987
1 6345 772 2498
2 4852 1105 1655
1 7048 1041 4416
2 7804 5761 6362
1 9090 4913 8655
2 6977 2271 240
1 8541 7772 6952
1 6865 5380 148
1 8814 8316 4698
1 8969 5152 7175
2 8536 512 1126
1 7907 1358 1595
2 9837 5272 3492
