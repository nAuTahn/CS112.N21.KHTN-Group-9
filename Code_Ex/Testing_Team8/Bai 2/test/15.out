2790
94
3070
5170
929
2456
4176
1458
4324
1458
4890
1382
3959
130
2661
438
975
1000
5460
1170
3219
2347
3544
38
5759
2088
1190
2519
286
3451
3337
1342
1159
1899
3390
5525
1469
147
907
2422
1062
213
1240
4622
4745
417
2512
2785
190
3051
510
1764
1972
4004
1743
745
292
1964
4328
4259
5787
186
4499
1058
4110
993
1409
1238
2510
6503
667
1036
1466
5559
1001
4704
476
780
43
2857
906
3759
1017
452
860
403
3492
3599
231
1310
845
296
42
3795
701
1242
3918
1157
974
3179
541
2260
1104
1992
5760
8025
239
2956
4
3726
1288
511
166
3491
